server/pulp/server/async/app.py

```diff
@@ -87,6 +84,13 @@ def initialize_worker(sender, instance, **kwargs):
         get_resource_manager_lock(sender)
 
 
+@worker_shutdown.connect
+def shutdown_worker(signal, sender):
+    msg = _("Worker '%s' shutdown" % sender.hostname)
+    _logger.info(msg)
```

Comment:
Since `_delete_worker()` is also called by `Scheduler.close()` if this is moved to an `else` branch on [the `if` statement that checks `normal_shutdown`](https://github.com/pulp/pulp/blob/master/server/pulp/server/async/tasks.py#L246) then this will log for celerybeat also which I think would be good.